docarray/index/backends/hnswlib.py

```diff
@@ -435,7 +435,7 @@ def _get_num_docs_hnsw(self) -> int:
         """
         first_index_key = next(iter(self._hnsw_indices))  # Get the first key
         first_hnsw_index = self._hnsw_indices[first_index_key]
-        hnsw_num_docs = first_hnsw_index.get_document_count()
+        hnsw_num_docs = first_hnsw_index.element_count()
```

Comment:
it is not a method, it is an attribute

Reply:
hey joan I've fixed this....can you please say me what is the issue again...so that the tests are failing...

Reply:
You can check what is faling here https://github.com/docarray/docarray/actions/runs/6003026515/job/16306992708?pr=1765

And try to run the tests urself

Reply:
Okey, I see that the test asserts the number of docs after deletion, and this HNSWlib element_count may not count deleted